static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -132,6 +139,31 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
     return this.metadata.features;
   }
 
+  renderTabs() {
+    // TODO: Convert to styled component
+    const {integrationSlug} = this.props.params;
+    const {organization} = this.props;
+    // TODO(cathy): remove feature check
+    const tabs =
+      integrationSlug === 'github' && organization.features.includes('pr-comment-bot')
```

Comment:
huh.
no way to make this more extensible?

Could we have a mapping of enum to features to include or something? 🤷
Or have the features list from the BE response help dictate this somehow?

Reply:
oh good point. a map would be good